### PR TITLE
Notes added.

### LINKED ISSUE #43: notes for bots
So in case you aren't testing a bot just yet but you want to leave a note so you or another moderator could see it in the feature.

How'd it work is simple. A moderator that noticed an issue would run `del,addnote <bot> <note>`  which would add a note to that bot, and later, when (another) moderator is testing the given bot they can execute `del,notes <bot>` to see all the notes added and a moderator who added the notes.

In case that note is useless a moderator could run `del,removenote <bot> <noteid>` which would obviously remove the note from the provided bot.

I haven't mentioned yet, but - this would only be available for bots that haven't been approved yet and usable only by moderators/assistants/administrators. (All the notes would automatically get removed whenever the bot is approved)

### PR DESCRIPTION
**Pull request by xPolar**
Upon request, I have created the notes command; I haven't been able to test it; however, I trust the code will work.
There was no formatting documentation, so I tried to stick to the formatting of the other commands.
The entire bot's code is somewhat messy (No offense), as there aren't descriptive comments or docstrings.

This PR should close #43.

### CODE REVIEW

**Review by Cairo2k18 — COMMENTED** [review]
Looks good, I'll do some proper testing later on. Thank you!

File: cogs/utility.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 import datetime
 from time import monotonic
-from typing import List
+from typing import Union
```
Comment (TheMoksej) [inline cogs/utility.py:24]: may I ask why this is being imported?

Comment (xPolar) [reply cogs/utility.py:24]: I had planned to use Union to allow for multiple versions of a variable, typing,Union tries to turn the argument into different types, I can do Union[discord.Member, discord.User, int], and then it'll try and turn it into a member, if that fails, turn into User, if that fails turn into an int which I can then turn into a user by fetching the user. I then realized we don't need anything from any of those types, and int would be just fine. I then forgot to remove the import.

Comment (TheMoksej) [reply cogs/utility.py:24]: Yeah I know what Union is, I've been using it as well for my bot. And alright. Everything seems to be fine except line 347. You forgot to add `name=` as a positional argument for embed.add_field()

Comment (xPolar) [reply cogs/utility.py]: Oh yup, just fixed both issues!

**Review by TheMoksej — APPROVED** [review]

**Comment by TheMoksej** [comment]
Alright everything from syntax side should be good

**Comment by xPolar** [comment]
No problem, hope you guys enjoy the notes functionality.

**Review by Cairo2k18 — APPROVED** [review]
Looks good, I'm going to make some slight changes in an individual commit after this.

**Pull request merged**